src/tooltip/tooltip.js

```diff
@@ -221,7 +221,7 @@ angular.module('ui.bootstrap.tooltip', ['ui.bootstrap.position'])
 
               // And show the tooltip.
               ttScope.isOpen = true;
-              if (isOpenExp) {
+              if (isOpenExp && angular.isFunction(isOpenExp.assign)) {
```

Comment:
It will throw an exception if using an expression instead of a model value, this change will allow for both implementations.